### PR TITLE
Order attribute in documentation on FareTableColumn and FareTableRow missing

### PR DESCRIPTION
**Pull request by ue71603**
https://github.com/NeTEx-CEN/NeTEx/issues/476

### CODE REVIEW

File: xsd/netex_part_3/part3_fares/netex_fareTable_version.xsd [hunk 1]
```diff
@@ -91,8 +91,7 @@
 					<Requires>http://www.netex.org.uk/schemas/1.0/PATH/netex_prereqfile.xsd</Requires>
 				</Relation>
 				<Rights>Unclassified
-					 <Copyright>CEN, Crown Copyright 2009-2020</Copyright>
-				</Rights>
+					 <Copyright>CEN, Crown Copyright 2009-2020</Copyright></Rights>
```
Comment (skinkie) [inline xsd/netex_part_3/part3_fares/netex_fareTable_version.xsd:94]: Try to avoid changing unrelated stuff.

**Review by skinkie — APPROVED** [review]

**Comment by skinkie** [comment]
Question: isn't this a fix for master too?

**Comment by ue71603** [comment]
@skinkie as it was labeled 2.0 I did it for next.  Nobody complained so far :-)  @Aurige  your call.

File: xsd/netex_part_3/part3_fares/netex_fareTable_version.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/netex_part_3/part3_fares/netex_fareTable_version.xsd:94]: XMLSpy did do it automatically. Sorry, that I missed it.

**Review by skinkie — APPROVED** [review]

**Review by trurlurl — APPROVED** [review]
